cmd/crates/stellar-ledger/src/transport_zemu_http.rs

```diff
@@ -0,0 +1,127 @@
+// I was getting an error when trying to install this as a crate, so i'm just going to take the pieces that i need an put them here since i am not using the gRpc transport right now anyway
```

Comment:
I tried to use the TransportZemuHttp transport from [https://github.com/zondax/ledger-rs](https://github.com/zondax/ledger-rs) but was running into errors with their grpc transport, and making sure I had the right protobuf library installed. We're not using the grpc transport, so I just included the HTTP transport that I needed here.

This is being used as a transport for interacting with the speculos emulator in the tests, so I think we should be safe to include our own version of this file here for now.